api/client/client_test.go

```diff
@@ -372,7 +372,7 @@ func TestClientMultipartUpload(t *testing.T) {
 
 	// check the tag was created successfully
 	tag := srv.Tags.All()[0]
-	chunktesting.CheckTag(t, tag, 9, 9, 0, 9)
+	chunktesting.CheckTag(t, tag, 8, 8, 0, 0, 0, 8)
```

Comment:
we should check why the modified use of context in http/server results in different number of tags. a manifest is missing maybe?
@zelig